doc/app/SOCKET_AT_commands.rst

```diff
@@ -1453,6 +1461,226 @@ Example
 
    OK
 
+Configure socket mode #XSOCKETMODE
+==================================
+
+The socket mode command allows you to configure the following aspects of a socket:
+
+* Automatic data reception
+* Automatic data reception in hex format
+
+Set command
+-----------
+
+The set command allows you to configure the socket mode of all sockets or a specific socket.
+
+Syntax
+~~~~~~
+
+::
+
+   AT#XSOCKETMODE=[<handle>],<auto_reception_flags>[,<hex_format>]
+
+* The ``<handle>`` parameter is an integer that identifies the socket handle.
+  If omitted, the command applies to all opened sockets.
```

Comment:
Just provoking discussion. For `#XAPOLL`, empty handle means it applies to all opened and those that will be opened in the future, right?
If yes, should we align here?
I'm not 100% sure about this but something we should think about.